src/components/presentational/CalendarDay/CalendarDay.stories.tsx

```diff
@@ -0,0 +1,254 @@
+import React, { useContext } from 'react';
+import CalendarDay, { CalendarDayProps, CalendarDayStateConfiguration } from './CalendarDay';
+import { Calendar, Card, DateRangeContext, DateRangeCoordinator, Layout } from '../../presentational';
+import { add, formatISO, isSameDay, parseISO } from 'date-fns';
+
+export default {
+    title: 'Presentational/CalendarDay',
+    component: CalendarDay,
+    parameters: {layout: 'fullscreen'}
+};
+
+const render = (args: CalendarDayProps) => <Layout colorScheme="auto">
+    <div style={{width: '60px', margin: '0 auto'}}>
+        <CalendarDay {...args} />
+    </div>
+</Layout>;
+
+const now = parseISO('2023-11-29T21:41:43.678Z');
```

Comment:
nit: use today

Reply:
I renamed it to `testDate`.  It doesn't really represent "now" or "today".  It used to be `new Date()`, which worked for "now", but I had to set it to a fixed date.  Otherwise, depending on which day of the week/month it was, the story renderings would be different.  Now they are static.